tests/test_aes.py

```diff
@@ -16,3 +20,41 @@ def test_encrypt_decrypt():
     encrypted_msg = encrypt(key_, msg_, 0)
     txt = decrypt(key_, encrypted_msg, 0)
     assert txt == msg_
+
+
+def test_AEAD_good():
+    key = os.urandom(32)
```

Comment:
Do you want to take advantage of our new and shiny `conftest.py` and add what you can (seed, iv, cleartext) as fixtures?